src/App.js

```diff
@@ -1,11 +1,37 @@
-import logo from "./logo.svg";
 import "./App.css";
+import Home from "./components/home/Home";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import Login from "./components/login/Login";
+import { UserProvider } from "./components/context/UserContext";
+import ProfileCreate from "./views/profileCreate/ProfileCreate";
+import EditProfile from "./views/profile/EditProfile";
+import Header from "./components/header/Header";
+import PrivateRoute from "./components/privacy/PrivateRoute";
+import { ProfileProvider } from "./components/context/ProfileContext";
 
```

Comment:
The context folder is tradtionally in src. It should be a sibiling of components instead of a child of components.

Also Login and Home are views instead of components. View essentially are pages of an app.